### PR TITLE
[Feature] Support imagenet pre-training for RTMDet's backbone.

### PR DESCRIPTION
**Pull request by RangiLyu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Provide pre-training backbone weights and configs.

|    Model     | resolution | Params(M) | Flops(G) | Top-1 (%) | Top-5 (%) |                                                             Download                                                             |
| :----------: | :--------: | :-------: | :------: | :-------: | :-------: | :------------------------------------------------------------------------------------------------------------------------------: |
| CSPNeXt-tiny |  224x224   |   2.73    |  0.339   |   69.44   |   89.45   | [model](https://download.openmmlab.com/mmdetection/v3.0/rtmdet/cspnext_rsb_pretrain/cspnext-tiny_imagenet_600e.pth) |
|  CSPNeXt-s   |  224x224   |   4.89    |  0.664   |   74.41   |   92.23   |  [model](https://download.openmmlab.com/mmdetection/v3.0/rtmdet/cspnext_rsb_pretrain/cspnext-s_imagenet_600e.pth)   |

### CODE REVIEW

File: configs/rtmdet/cspnext_imagenet_pretrain/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# CSPNeXt ImageNet Pre-training
+
+In this folder, we provide the imagenet pre-training config of RTMDet's backbone CSPNeXt.
+To train with these configs, please install [MMClassification](https://github.com/open-mmlab/mmclassification) first.
```
Comment (hhaAndroid) [inline configs/rtmdet/cspnext_imagenet_pretrain/README.md:4]: Which specific branch is it?

Comment (hhaAndroid) [reply configs/rtmdet/cspnext_imagenet_pretrain/README.md:4]: Please provide a command that can be installed directly

File: configs/rtmdet/cspnext_imagenet_pretrain/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## Results and Models
+
+|    Model     | resolution | Params(M) | Flops(G) | Top-1 (%) | Top-5 (%) |                                                             Download                                                             |
+| :----------: | :--------: | :-------: | :------: | :-------: | :-------: | :------------------------------------------------------------------------------------------------------------------------------: |
+| CSPNeXt-tiny |  224x224   |   2.73    |  0.339   |   69.44   |   89.45   | [model](https://download.openmmlab.com/mmdetection/v3.0/rtmdet/cspnext_rsb_pretrain/cspnext_tiny/cspnext-tiny_imagenet_600e.pth) |
```
Comment (hhaAndroid) [inline configs/rtmdet/cspnext_imagenet_pretrain/README.md:10]: No Log?

File: configs/rtmdet/cspnext_imagenet_pretrain/cspnext-s_8xb256-rsb-a1-600e_in1k.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
+_base_ = [
```
Comment (hhaAndroid) [inline configs/rtmdet/cspnext_imagenet_pretrain/cspnext-s_8xb256-rsb-a1-600e_in1k.py:1]: How to start training and testing?

File: configs/rtmdet/cspnext_imagenet_pretrain/README.md (hunk 2 above)
Comment (RangiLyu) [reply configs/rtmdet/cspnext_imagenet_pretrain/README.md:35]: yes

File: configs/rtmdet/cspnext_imagenet_pretrain/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# CSPNeXt ImageNet Pre-training
+
+In this folder, we provide the imagenet pre-training config of RTMDet's backbone CSPNeXt.
+
+## Requirements
+
+To train with these configs, please install [MMClassification-1.x](https://github.com/open-mmlab/mmclassification/tree/1.x) first.
```
Comment (ZwwWayne) [inline configs/rtmdet/cspnext_imagenet_pretrain/README.md:7]: MMClassification-1.x -> MMClassification 1.x

File: configs/rtmdet/cspnext_imagenet_pretrain/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## Prepare Dataset
+
+To pre-train on ImageNet, you need to prepare the dataset first. Please refer to the [guide](https://mmclassification.readthedocs.io/en/1.x/user_guides/dataset_prepare.html#imagenet).
+
+## How to Train
+
+You can use the classification config in the same way as the detection config.
+
+More details can be found in [user guides](https://mmdetection.readthedocs.io/en/3.x/user_guides/train.html).
```
Comment (ZwwWayne) [inline configs/rtmdet/cspnext_imagenet_pretrain/README.md:29]: Simply give a command here to make this readme self-contained, i.e., experienced users can easily train by reading this doc only.

**Review by hhaAndroid — APPROVED** [review]

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**